packages/api/server/http.mts

```diff
@@ -104,17 +87,24 @@ router.delete('/srcbooks/:id', cors(), async (req, res) => {
 // Import a srcbook from a .src.md file or srcmd text.
 router.options('/import', cors());
 router.post('/import', cors(), async (req, res) => {
-  const { path, text } = req.body;
+  const { path, text, url } = req.body;
 
   if (typeof path === 'string' && !isSrcmdPath(path)) {
     return res.json({ error: true, result: 'Importing only works with .src.md files' });
   }
+  if (typeof url === 'string' && !isSrcmdPath(url)) {
```

Comment:
I think it's more robust to make no assumptions about paths. For now the hub will have this format, but that may not always be the case. More importantly, in theory, these could be hosted or available anywhere by anyone (even though today that's very unlikely)

Reply:
The thing that matters is that we fail if we cannot successfully parse it, which I think is still happening?

Reply:
Ok, sounds good!

> The thing that matters is that we fail if we cannot successfully parse it, which I think is still happening?
Yep, that is still happening!